### PR TITLE
MD2 (Microdiff) bug fixes and improvements

### PR DESCRIPTION
**Pull request by mguijarr**

### CODE REVIEW

File: HardwareObjects/Microdiff.py [hunk 1]
```diff
@@ -53,17 +53,20 @@ def init(self):
...
         self.centringFocus = self.getDeviceByRole('centringFocus')   
         
-        self.frontLight = self.getDeviceByRole('flight')
-        self.backLight = self.getDeviceByRole('light')  
+        self.frontLight = self.getDeviceByRole('FrontLight')
+        self.backLight = self.getDeviceByRole('BackLight')  
+        self.beam_info = self.getObjectByRole('beam_info')
+
+        self.frontLight = self.getDeviceByRole('FrontLight')
+        self.backLight = self.getDeviceByRole('BackLight')
```
Comment (meguiraun) [inline HardwareObjects/Microdiff.py]: repeated lines

Comment (mguijarr) [reply HardwareObjects/Microdiff.py]: Good to see someone is looking at this ;)

**Comment by meguiraun** [comment]
did you do a rebase or something? because now I see 14 commits in this pr

**Comment by mguijarr** [comment]
Yeah @meguiraun I messed up with git ;) Not the first time. Reparing...

**Comment by meguiraun** [comment]
my daily life :( ...

**Comment by mguijarr** [comment]
Voila, should be fine now.

**Pull request merged**